iam/src/providers/facebookProfilePicture.ts

```diff
@@ -0,0 +1,76 @@
+// ----- Types
+import type { Provider, ProviderOptions } from "../types";
+import type { RequestPayload, VerifiedPayload } from "@gitcoin/passport-types";
+
+// --- Api Library
+import axios from "axios";
+
+export type FacebookProfileResponse = {
+  id: string;
+  picture: {
+    data: {
+      is_silhouette: boolean;
+    };
+  };
+};
+
+// Facebook Graph API call response
+type Response = {
+  data?: FacebookProfileResponse;
+  status?: number;
+  statusText?: string;
+  headers?: {
+    [key: string]: string;
+  };
+};
+
+// Query Facebook graph api to verify the profile picture
+export class FacebookProfilePictureProvider implements Provider {
+  // Give the provider a type so that we can select it with a payload
+  type = "FacebookProfilePicture";
+  // Options can be set here and/or via the constructor
+  _options = {};
+
+  // construct the provider instance with supplied options
+  constructor(options: ProviderOptions = {}) {
+    this._options = { ...this._options, ...options };
+  }
+
+  // verify that the proof object contains valid === "true"
+  async verify(payload: RequestPayload): Promise<VerifiedPayload> {
+    try {
+      // Get the FB profile
```

Comment:
@gdixon Do you think we should validate the `payload.proofs.accessToken` similar like we do in `facebookFriends.ts`?
There is no real reason to do this for this stamp, except for additional security, but I am not sure about this.

Reply:
After checking out the stamp and looking at the additional access token verification I think it is good to keep it. There does not seem to be a performance impact from it